src/asset-management/AaveManager.sol

```diff
@@ -70,11 +70,11 @@ contract AaveManager is IAssetManager, Ownable, ReentrancyGuard
         IAssetManagedPair aPair,
         int256 aAmount0Change,
         int256 aAmount1Change
-    ) external nonReentrant onlyOwner {
+    ) external onlyOwner {
         _adjustManagement(aPair, aAmount0Change, aAmount1Change);
     }
 
-    function _adjustManagement(IAssetManagedPair aPair, int256 aAmount0Change, int256 aAmount1Change) private {
+    function _adjustManagement(IAssetManagedPair aPair, int256 aAmount0Change, int256 aAmount1Change) nonReentrant private {
```

Comment:
lint

Reply:
Fixed. Oh yeah wanted to highlight this. I think it's more appropriate to move the reentrancy guard to this internal function as it can be called by multiple external function and is the "heart" of the asset manager. Whereas the external `adjustManagement` function is only just a shell

Reply:
Makes sense, so long as its never called twice legitimately (doesnt seem like that would ever make sense?)

Reply:
yup, can't think of a scenario where this is possible